graphium/finetuning/finetuning.py

```diff
@@ -29,7 +29,8 @@ def __init__(
         finetuning_module: str,
         added_depth: int = 0,
         unfreeze_pretrained_depth: Optional[int] = None,
-        epoch_unfreeze_all: int = 0,
+        epoch_unfreeze_all: Optional[int] = 0,
+        freeze_always: Optional[Union[List, str]] = None,
```

Comment:
`freeze_always` -> `always_freeze_modules`

Also docstring is missing for that variable, It think it's important